optuna/samplers/cmaes.py

```diff
@@ -324,6 +324,10 @@ def _initialize_x0(search_space: Dict[str, BaseDistribution]) -> Dict[str, np.nd
             log_high = math.log(distribution.high)
```

Comment:
Sorry for the confusion. If you raise `NotImplementedError`, my comments on `CmaEsSampler` can be ignored. (But, it can be resolved by my comments.)

Reply:
@HideakiImamura Actually, I proposed @nzw0301 to put a todo comment because it seems to be hard to support CmaEsSampler in this PR (https://github.com/optuna/optuna/pull/1201#issuecomment-629960251). What do you think?

Reply:
[IMO] I think it's enough just put a todo comment because it will be just fallbacked to independent sampler with warning messages.

Reply:
I agree with c-bata-san's opinion. For the `CmaEsSampler`, we do not need to raise `NotImplementedErroe`. Thank you for your pointing.

Reply:
I keep the changes and update them regarding @HideakiImamura 's comments.